client/src/commons/Contexts/PersonalInfoStateContext.ts

```diff
@@ -5,9 +5,9 @@ import {personalInfoContextData} from 'models/Contexts/personalInfoContextData';
 import { initialAddress } from './ClinicalDetailsContext';
 
 export const initialPersonalInfo: personalInfoContextData = {
-    phoneNumber: '',
-    additionalPhoneNumber: '',
-    contactPhoneNumber: '',
+    phoneNumber: {number:'', isValid: true},
+    additionalPhoneNumber: {number:'', isValid: true},
+    contactPhoneNumber: {number:'', isValid: true},
```

Comment:
should it be valid by default despite it being an empty value?

Reply:
i think so

Reply:
maybe not for mandatory fields